roles/jenkins_common/tasks/main.yml

```diff
@@ -1,28 +1,30 @@
 - name: install beaker client
+  become: true
```

Comment:
I like to remove as much repetition from playbooks as possible. I'd prefer to see this at the higher level. If the level of efort is significant, let's open an issue on it rather than do it here.